corehq/form_processor/models.py

```diff
@@ -88,4 +92,132 @@ def read_content(self):
             content = f.read()
         return content
 
-Attachment = collections.namedtuple('Attachment', 'name content content_type')
+
+class CommCareCaseSQL(models.Model, AbstractCommCareCase, RedisLockableMixIn):
+    case_uuid = models.CharField(max_length=255, unique=True, db_index=True)
+    domain = models.CharField(max_length=255)
+    case_type = models.CharField(max_length=255)
+
+    owner_id = models.CharField(max_length=255)
+
+    opened_on = models.DateTimeField(null=False)
+    opened_by = models.CharField(max_length=255, null=False)
+
+    modified_on = models.DateTimeField(null=False)
+    server_modified_on = models.DateTimeField(null=False)
+    modified_by = models.CharField(max_length=255)
+
+    closed = models.BooleanField(default=False, null=False)
+    closed_on = models.DateTimeField(null=True)
+    closed_by = models.CharField(max_length=255, null=False)
+
+    deleted = models.BooleanField(default=False, null=False)
+
+    external_id = models.CharField(max_length=255)
+
+    case_json = JSONField(lazy=True)
+    attachments_json = JSONField(lazy=True)
```

Comment:
Alternative approach to https://github.com/dimagi/commcare-hq/pull/8941/files#diff-08eddcfb85f88441d43dbfad3c1fac4dR104

Not sure which is better.